
[root@ip-10-19-155-28 container_1494008008691_0014_01_000001]# cat stdout 


[root@ip-10-19-155-28 container_1494008008691_0014_01_000001]# cat stderr 
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/mnt1/yarn/usercache/hadoop/filecache/67/__spark_libs__7135182162179608188.zip/slf4j-log4j12-1.7.16.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/usr/lib/hadoop/lib/slf4j-log4j12-1.7.10.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.impl.Log4jLoggerFactory]
17/05/05 21:34:40 INFO SignalUtils: Registered signal handler for TERM
17/05/05 21:34:40 INFO SignalUtils: Registered signal handler for HUP
17/05/05 21:34:40 INFO SignalUtils: Registered signal handler for INT
17/05/05 21:34:41 INFO ApplicationMaster: Preparing Local resources
17/05/05 21:34:42 INFO ApplicationMaster: ApplicationAttemptId: appattempt_1494008008691_0014_000001
17/05/05 21:34:42 INFO SecurityManager: Changing view acls to: yarn,hadoop
17/05/05 21:34:42 INFO SecurityManager: Changing modify acls to: yarn,hadoop
17/05/05 21:34:42 INFO SecurityManager: Changing view acls groups to: 
17/05/05 21:34:42 INFO SecurityManager: Changing modify acls groups to: 
17/05/05 21:34:42 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(yarn, hadoop); groups with view permissions: Set(); users  with modify permissions: Set(yarn, hadoop); groups with modify permissions: Set()
17/05/05 21:34:42 INFO ApplicationMaster: Starting the user application in a separate Thread
17/05/05 21:34:42 INFO ApplicationMaster: Waiting for spark context initialization...
17/05/05 21:34:43 ERROR ApplicationMaster: User application exited with status 255
17/05/05 21:34:43 INFO ApplicationMaster: Final app status: FAILED, exitCode: 255, (reason: User application exited with status 255)
17/05/05 21:34:43 ERROR ApplicationMaster: Uncaught exception: 
org.apache.spark.SparkException: Exception thrown in awaitResult: 
    at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:194)
    at org.apache.spark.deploy.yarn.ApplicationMaster.runDriver(ApplicationMaster.scala:401)
    at org.apache.spark.deploy.yarn.ApplicationMaster.run(ApplicationMaster.scala:254)
    at org.apache.spark.deploy.yarn.ApplicationMaster$$anonfun$main$1.apply$mcV$sp(ApplicationMaster.scala:766)
    at org.apache.spark.deploy.SparkHadoopUtil$$anon$1.run(SparkHadoopUtil.scala:67)
    at org.apache.spark.deploy.SparkHadoopUtil$$anon$1.run(SparkHadoopUtil.scala:66)
    at java.security.AccessController.doPrivileged(Native Method)
    at javax.security.auth.Subject.doAs(Subject.java:422)
    at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1698)
    at org.apache.spark.deploy.SparkHadoopUtil.runAsSparkUser(SparkHadoopUtil.scala:66)
    at org.apache.spark.deploy.yarn.ApplicationMaster$.main(ApplicationMaster.scala:764)
    at org.apache.spark.deploy.yarn.ApplicationMaster.main(ApplicationMaster.scala)
Caused by: org.apache.spark.SparkUserAppException: User application exited with 255
    at org.apache.spark.deploy.PythonRunner$.main(PythonRunner.scala:96)
    at org.apache.spark.deploy.PythonRunner.main(PythonRunner.scala)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.apache.spark.deploy.yarn.ApplicationMaster$$anon$2.run(ApplicationMaster.scala:637)
17/05/05 21:34:43 INFO ShutdownHookManager: Shutdown hook called
[root@ip-10-19-155-28 container_1494008008691_0014_01_000001]# 



